cmd/cluster-olm-operator/main.go

```diff
@@ -119,7 +128,7 @@ func runOperator(ctx context.Context, cc *controllercmd.ControllerContext) error
 
 	cl.StartInformers(ctx)
 
-	for _, c := range append(controllerList, upgradeableConditionController, clusterOperatorController) {
+	for _, c := range append(controllerList, staticUpgradeableConditionController, dynamicUpgradeableConditionController, clusterOperatorController) {
```

Comment:
I haven't dived in to look further, but it seems like it could be a problem to have two different controllers trying to control the upgradeable condition, right?

Reply:
Looked a little further the `statusUpgradeableConditionController` only seems to set `${prefix}Upgradeable` conditions (good!), but then what (if anything) is aggregating those up to a single `Upgradeable` condition?

Reply:
yeah I thought the update function would only update prefix specific types but after some testing this afternoon I realized it wasn't the case - moved that sync in the dynamicUpgradeableConditionController code